src/libs/db/schema.ts

```diff
@@ -66,3 +67,14 @@ export const verificationTokens = pgTable(
         },
     ],
 );
+
+export const profiles = pgTable("profile", {
+    profileID: text("profile_ID").notNull().primaryKey(),
+    userID: text("user_ID").notNull(),
```

Comment:
the profile should map to reference back to the user do

```suggestion
    userId: text('user_id').references(() => users.id),
```